packages/qdrant-loader/src/qdrant_loader/cli/commands/jobs_cmd.py

```
@@ -48,30 +51,23 @@ def _load_config(workspace: Path | None, config: Path | None, env: Path | None)
             skip_validation=True,
         )
 
+    global_config = get_global_config()
+    state_manager = StateManager(global_config.state_management)
+    return state_manager
+
 
 async def _run_with_queue(workspace, config, env, coro_factory):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_run_with_queue`

(ANN202)